server/src/browser-management/classes/RemoteBrowser.ts

```
@@ -427,6 +427,541 @@ export class RemoteBrowser {
+    
+                for (const element of currentElements) {
+                  if ((element as Element).tagName === 'IFRAME' || (element as Element).tagName === 'FRAME') {
+                    try {
+                      const frameElement = element as HTMLIFrameElement | HTMLFrameElement;
+                      const frameDoc = frameElement.contentDocument || frameElement.contentWindow?.document;
+                      if (frameDoc) {
+                        nextElements.push(...Array.from(frameDoc.querySelectorAll(part)));
+                      }
+                    } catch (e) {
+                      console.warn(`Cannot access ${(element as Element).tagName.toLowerCase()} content:`, e);
+                      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)